src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -885,6 +885,7 @@ def test_aks_upgrade_node_image_only_nodepool(self, resource_group, resource_gro
                                                '--cluster-name {name} ' \
                                                '-n {node_pool_name} ' \
                                                '--node-image-only ' \
+                                               '--aks-custom-headers fake-key1=value1,fake-key2=value2' \
```

Comment:
`--aks-custom-headers` is basically used for preview features thus a specific header is temporarily supported. Instead of using a specific header, we use a fake header and value, which is also supported but brings no actual effect on the command.
We can discuss here.

Reply:
cc @zhoxing-ms

Reply:
May I ask why don't you use the specific header here?

Reply:
@zhoxing-ms , I added a new test to do aks nodepool upgrade with the customer header, thanks for your advice.